src/pke/lib/scheme/bfvrns/bfvrns-parametergeneration.cpp

```diff
@@ -147,7 +160,7 @@ bool ParameterGenerationBFVRNS::ParamsGenBFVRNS(std::shared_ptr<CryptoParameters
     if ((multiplicativeDepth == 0) && (keySwitchCount == 0)) {
         // Correctness constraint
         auto logqBFV = [&](uint32_t n) -> double {
-            return std::log(p * (4 * ((evalAddCount + 1) * Vnorm(n) + evalAddCount) + p));
+            return std::log2(p * (4 * ((evalAddCount + 1) * Vnorm(n) + evalAddCount) + p));
```

Comment:
I agree with computing directly log2 rather than carrying log(2) everywhere else.

Reply:
Initially we used log/log(2) because an old version of Visual C++ did not support it. to the best of my knowledge, log2 is supported by all compilers OpenFHE currently supports.